app/controllers/agenda/agendaitems/agendaitem/index.js

```diff
@@ -64,7 +64,7 @@ export default class IndexAgendaitemAgendaitemsAgendaController extends Controll
     await this.navigateToNeighbouringItem(this.model);
     // reload the agenda route, detail tab should no longer show if we deleted the last and only agendaitem
     // Also, if we deleted the first agendaitem, we should also reload the main route to reload <Agenda::agendaTabs>
-    return this.agendaController.send('reloadAgendaModel');
```

Comment:
this was a parent route refresh for route `agenda`, changing to `this.router.refresh('agenda');`